config/locales/cy.yml

```
@@ -466,6 +466,8 @@ cy:
   preview_banner:
     heading: This is a preview of this form. Do not enter personal information.
     title: Important
+  preview_header:
+    your_questions: ''
```

This can be 'Your questions' - it's content for form creators so we don't need to translate it:

```suggestion
    your_questions: Your questions
```